.github/workflows/ci.yml

```diff
@@ -51,7 +50,6 @@ jobs:
         run: ./ci/script.sh
 
       - name: Cache dependencies
-        if: ${{ success() }}
```

Comment:
Rather than remove this, the build step should probably set a variable that we can check.  If the build succeeds, then caching the dependencies is ok.